File: win/SpotCafeClientComponents/Desktop/SpotCafe.Desktop/WebSocketMessageName.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SpotCafe.Desktop {
    public static class WebSocketMessageName {
        public const string Ping = "ping";
        public const string GetDrivesRequest = "get-drives-request";
        public const string GetDrivesResponse = "get-drives-response";
        public const string GetFolderItemsRequest = "get-folder-items-request";
        public const string GetFolderItemsResponse = "get-folder-items-response";
        public const string StartDevice = "start-device";
        public const string StopDevice = "stop-device";
        public const string GetProcessesRequest = "get-processes-request";
        public const string GetProcessesResponse = "get-processes-response";
        public const string KillProcessRequest = "kill-process-request";
        public const string KillProcessResponse = "kill-process-response";
        public const string ExecuteActionRequest = "execute-action-request";
    }

    public static class WebSocketError {
        public const int CantGetProcesses = 1;
        public const int CantKillProcess = 2;
    }
}

File: shared/web-socket-message-name.ts
export enum WebSocketMessageName {
    ping = 'ping',
    getDrivesRequest = 'get-drives-request',
    getDrivesResponse = 'get-drives-response',
    getFolderItemsRequest = 'get-folder-items-request',
    getFolderItemsResponse = 'get-folder-items-response',
    startDevice = 'start-device',
    stopDevice = 'stop-device',
    getProcessesRequest = 'get-processes-request',
    getProcessesResponse = 'get-processes-response',
    killProcessRequest = 'kill-process-request',
    deviceNotFoundError = 'device-not-found-error',
    executeActionRequest = 'execute-action-request'
}

export enum WebSocketMessageErrorNumber {
    deviceNotConnected = 1
}
